src/main/java/seedu/mindmymoney/command/ListCommand.java

```diff
@@ -8,23 +9,28 @@
     public ListCommand() {
     }
 
+    public String listToString() {
+        Integer indexOfList = 1;
```

Comment:
```suggestion
        int indexOfList = 1;
```
Maybe just use int to be consistent with the rest of the code